new-frontiers-14/frontier-station-14
# Extends Halloween to one week

## About the PR
This PR changes the start and end date for Halloween to October 27th to November 2nd. It also pushes back the start date of Festive Season to November 3rd to avoid excessive holiday overlaps between it, Halloween, and Vegan Day on November 1st-2nd.

## Why / Balance
The recent cauldron PR https://github.com/new-frontiers-14/frontier-station-14/pull/2240 added the first seasonal Halloween object variant to the game and created the groundwork for future ones to be added. It would therefore be beneficial going forwards if these seasonal variants were available in-game for more than one day to ensure that the vast majority of players had the opportunity to enjoy the seasonal content. This PR specifically extends Halloween to be four days before the 31st and two days afterwards to ensure that this content is accessible for individuals with varying schedules regardless of what week day Halloween lands on in future years.

## How to test
1. Set your computer's clock to any date from October 27th to November 2nd.
2. Run your copy of Frontier and start the round. You should receive the "Have a Spooky Halloween!" lobby message.
3. Spawn on the dev map, buy a food ship, or visit a food POI. The deep fryer should now be its seasonal variant.
4. Set your computer's clock back to its regular time.
5. Run your copy of Frontier and start the round. You should not receive the Halloween lobby message and the previous steps should return their non-seasonal forms.

## Media
N/A

## Requirements


## Breaking changes
None.

**Changelog**
:cl:
- tweak: Halloween now runs from Oct. 27-November 2nd.
